pushdeploy/apiv1.py

```diff
@@ -74,12 +106,62 @@ def login():
     access_token = create_access_token(identity=username, expires_delta=timedelta(seconds=90))
     return jsonify(access_token=access_token), 200
 
+@bp.route('/cronjob', methods=['POST'])
+@jwt_required
+def cronjob():
+    image_tag = request.args['image_tag']
+    image_name = request.args['image_name']
+    name = request.args['name']
+    namespace = request.args['namespace']
+    cronjob = patch_cron_job(
+        cron_job_object=list_cron_job(name=name, namespace=namespace),
```

Comment:
`patch_cron_job` should probably be responsible for fetching the `cron_job_object`. It's signature already takes a `name` & `namespace` so you wouldn't have to add anything, but you would be able to remove something and simplify the call sites.